test/client.tests.js

```diff
@@ -3,27 +3,31 @@ const _ = require('lodash');
 const assert = require('chai').assert;
 const LimitRedis = require('../lib/client');
 const ValidationError = LimitRedis.ValidationError;
+const clusterNodes = [{ host: '127.0.0.1', port: 16371 }, { host: '127.0.0.1', port: 16372 }, { host: '127.0.0.1', port: 16373 }];
 
-describe('LimitdRedis', () => {
+const standaloneClientFn = (params) => {
+  return new LimitRedis({ uri: 'localhost', buckets: {}, prefix: 'tests:', ..._.omit(params, ['nodes']) });
+};
+const clusteredClientFn = (params) => {
+  return new LimitRedis({ nodes: clusterNodes, buckets: {}, prefix: 'tests:', ..._.omit(params, ['uri']) });
+};
+let clientCreator = standaloneClientFn;
+let clusteredEnv = false;
+
+if (process.env.CLUSTERED_ENV && process.env.CLUSTERED_ENV === "true") {
```

Comment:
I feel like we should have either two tests or two describe blocks, rather than have an env var which then does conditional logic in the test

Reply:
totally true. I rearranged the tests to make it explicit. Thanks for the suggestion